drivers.xml

```diff
@@ -949,6 +953,10 @@
             <driver name="Uranus Meteo Sensor">indi_uranus_weather</driver>
             <version>1.0</version>
         </device>
+        <device label="Weather Safety Alpaca" manufacturer="Others">
```

Comment:
Maybe change manufacturer to Alpaca? I know it's not technically a manufacturer but it would be nice grouping for all future alpaca-compatible drivers.

Reply:
Perfect, I'll do that :)

Reply:
Done, i have changed for both :)